tests/test_fixtures/rule_based_profiler/example_notebooks/DataAssistants_Instantiation_And_Running-OnboardingAssistant-Spark.ipynb

```diff
@@ -0,0 +1,1498 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "8362d4be",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/work/Development/ENVs/supercon_ge/lib/python3.8/site-packages/snowflake/connector/options.py:94: UserWarning: You have an incompatible version of 'pyarrow' installed (7.0.0), please install a version that adheres to: 'pyarrow<3.1.0,>=3.0.0; extra == \"pandas\"'\n",
+      "  warn_incompatible_dep(\n"
+     ]
+    }
+   ],
+   "source": [
+    "import great_expectations as ge\n",
+    "from great_expectations.core.batch import BatchRequest\n",
+    "from great_expectations.core import ExpectationSuite\n",
+    "from great_expectations.core.expectation_configuration import ExpectationConfiguration\n",
+    "from great_expectations.validator.validator import Validator\n",
+    "from great_expectations.rule_based_profiler.data_assistant import (\n",
+    "    DataAssistant,\n",
+    "    VolumeDataAssistant,\n",
+    ")\n",
+    "from great_expectations.rule_based_profiler.data_assistant_result import (\n",
+    "    VolumeDataAssistantResult,\n",
+    ")\n",
+    "from typing import List\n",
+    "from ruamel import yaml\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1df5289b",
+   "metadata": {},
+   "source": [
+    "### Example Directory\n",
+    "\n",
+    "Imagine we have a directory of 24 csv files, each corresponding to 1 month of NYC taxi rider data from 2019 to 2020.\n",
+    "\n",
+    "```\n",
+    "\n",
+    "yellow_tripdata_sample_2019-01.csv\n",
+    "yellow_tripdata_sample_2019-02.csv\n",
+    "yellow_tripdata_sample_2019-03.csv\n",
+    "yellow_tripdata_sample_2019-04.csv\n",
+    "yellow_tripdata_sample_2019-05.csv\n",
+    "yellow_tripdata_sample_2019-06.csv\n",
+    "yellow_tripdata_sample_2019-07.csv\n",
+    "yellow_tripdata_sample_2019-08.csv\n",
+    "yellow_tripdata_sample_2019-09.csv\n",
+    "yellow_tripdata_sample_2019-10.csv\n",
+    "yellow_tripdata_sample_2019-11.csv\n",
+    "yellow_tripdata_sample_2019-12.csv\n",
+    "yellow_tripdata_sample_2020-01.csv\n",
+    "yellow_tripdata_sample_2020-02.csv\n",
+    "yellow_tripdata_sample_2020-03.csv\n",
+    "yellow_tripdata_sample_2020-04.csv\n",
+    "yellow_tripdata_sample_2020-05.csv\n",
+    "yellow_tripdata_sample_2020-06.csv\n",
+    "yellow_tripdata_sample_2020-07.csv\n",
+    "yellow_tripdata_sample_2020-08.csv\n",
+    "yellow_tripdata_sample_2020-09.csv\n",
+    "yellow_tripdata_sample_2020-10.csv\n",
+    "yellow_tripdata_sample_2020-11.csv\n",
+    "yellow_tripdata_sample_2020-12.csv\n",
+    "\n",
+    "```\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2fdc0ab6",
+   "metadata": {},
+   "source": [
+    "## Set-up: Importing `pyspark`"
```

Comment:
This notebook isn't intended to be a general Doc, but more of a "bridge" between the code and the documentation, since we have separate notebooks for the different backends (sql and pandas), would it be ok to keep this as-is?

Reply:
Sounds good. Do you want to update the title of the doc to say spark? Whatever you think is most appropriate is good with me.